app/middleware/createGpViewModel.js

```diff
@@ -29,6 +30,7 @@ function createGpViewModel(req, res, next) {
       choicesId: gpData.choicesId,
       location: gpData.location,
       facilities: parseFacilities(gpData.facilities),
+      services: servicesMapper(gpData.services),
```

Comment:
Totally agree, some consistency would be best and I've broken it here. My preference would be `mappers` and appreciate the argument for keeping to whatever is already in the code (in this case `parseXXX`). Worth the discussion now though.

@neilbmclaughlin Care to add your thoughts on this?